src/Webserver/MinkConfiguration.php

```diff
@@ -20,12 +20,17 @@
      * @var
      */
     private $docroot;
+    /**
+     * @var
+     */
+    private $router;
 
-    public function __construct($host, $port, $docroot, $baseUrl)
+    public function __construct($host, $port, $docroot, $router, $baseUrl)
```

Comment:
Yeah I constructor signatures are a grey area but to keep SemVer I'd rather not break it.